### DISCUSSION TITLE
Display organizations on the home page

### DISCUSSION THREAD

**Discussion by douglaslassance** [community/community#6561] [Other Feature Feedback, Questions, & Ideas]
I find it always weirdly contrived to access my organizations from the [GitHub's home page](https://github.com/). It usually involves clicking on a random repository and then clicking on the organization link from the repository page. Sometimes the repositories of the organization I am interested in are buried in "show more" which takes an extra click and scroll. I think that adding links to the organizations you belong to on the home page would greatly improve the site's flow.

**Comment by kevinamstutz** [comment]
I agree completely. I wish there were just a menu of organizations to click on to jump to that organizations repository page. Personally, I almost never care about the "feed" and if I do, I can see the activity in the organizations page or individual repositories page. But to get to an organization from the home page usually involves: 

1. accidentally clicking the "View organization" page instead of the dropdown menu where the organizations are listed.
2. navigating back to the home page to click on the menu to select the organization.
3. once the organization "feed" page loads, click on the button to "view organization". 

It may just be me, but I do that constantly. Maybe it would be good to move "Feed" to a tab in the organization's home page and just adapt the organization home pages to be the "dashboard" instead?
